examples/getting-started-app/README.md

```diff
@@ -481,9 +486,70 @@ If this value is unset, the failover status won't be affected.
 
 ### Stateful failover
 
-The full example for [stateful failover](https://www.tarantool.io/en/doc/2.2/book/cartridge/cartridge_api/topics/failover.md#stateful-failover)
-is coming soon.
-Now you can [read the doc](../../README.md#stateful).
+**Note** that stateful failover is supported since `Cartridge` 2.1.0.
```

Comment:
AFAIK 2.1.0 is removed from rocks.tarantool.org. Since 2.1.1

Reply:
Yes, 2.1.2, thanks!